api/app.py

```diff
@@ -1,46 +1,7 @@
-import os
-
-from flask import Flask
```

Comment:
I moved a bunch of stuff out of this file and into `api/config.py`. Basically, the idea is that `api/config.py` should have all the configuration for the API, and be usable in e.g. tests.